portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,10 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+async function getName() {
+    const response = await fetch('/data');
+      console.log('Handling the response.');
```

Comment:
nit: please make the indentation of this log line consistent with the other lines in the function.